solidity/contracts/bridge/BridgeState.sol

```diff
@@ -175,11 +175,14 @@ library BridgeState {
         // Value in seconds.
         uint32 walletCreationPeriod;
         // The minimum BTC threshold in satoshi that is used to decide about
-        // wallet creation or closing.
-        uint64 walletMinBtcBalance;
+        // wallet creation.
+        uint64 walletCreationMinBtcBalance;
         // The maximum BTC threshold in satoshi that is used to decide about
         // wallet creation.
-        uint64 walletMaxBtcBalance;
+        uint64 walletCreationMaxBtcBalance;
+        // The minimum BTC threshold in satoshi that is used to decide about
+        // wallet closing.
+        uint64 walletClosureMinBtcBalance;
```

Comment:
I would give these three parameters a more precise description as how they decide about wallet creation or closure, similar to how they are described in
[RFC](https://github.com/keep-network/tbtc-v2/blob/main/docs/rfc/rfc-1.adoc#wallet-lifecycle):
```
wallet_min_closure_btc: The smallest amount of btc a wallet can hold before we attempt to close the wallet and transfer the funds to a randomly selected wallet.

```